src/app/api/services/dot-apps/dot-apps.service.spec.ts

```diff
@@ -127,6 +131,33 @@ describe('DotAppsService', () => {
         expect(dotHttpErrorManagerService.handle).toHaveBeenCalledWith(mockResponseView(400));
     });
 
+    it('should import apps', () => {
+        spyOn(coreWebService, 'requestView').and.callThrough();
+        const conf: DotAppsImportConfiguration = {
+            file: null,
+            json: { password: 'test' }
+        };
+        const sentBody = new FormData();
+        sentBody.append('json', JSON.stringify(conf.json));
+        sentBody.append('file', conf.file);
+
+        dotAppsService.importConfiguration(conf).subscribe((status: string) => {
+            expect(status).toEqual('OK');
+        });
+
+        const req = httpMock.expectOne(`/api/v1/apps/import`);
```

Comment:
I prefer to use it, to get a response https://github.com/dotCMS/core-web/blob/48a565b472ca2e3e61ee66ba6347167428755f19/src/app/api/services/dot-apps/dot-apps.service.spec.ts#L157

otherwise I'll have to spyOn the response of coreWebService.requestView, which would be more complicated